migrations/4_genesis.js

```diff
@@ -73,5 +74,14 @@ module.exports = function(deployer, network, accounts) {
         deployer.logger.log("Ending genesis and transferring ownership of the LivepeerToken to the protocol Minter...")
 
         await genesisManager.end()
+
+        if (!lpDeployer.isProduction(network)) {
+            // If not in production, the deployment account holds the company supply
+            // and use some of those tokens to fund the faucet supply
+            deployer.logger.log("Not in production - deployment account will fund the deployed faucet")
+
+            const faucetAddr = await controller.getContract(contractId("LivepeerTokenFaucet"))
```

Comment:
If we are not deploying to main net and we are not doing a dry run on a test network, we have already sent the company supply to the deployment account at this point. So, the deployment account can use those tokens to fund the faucet which will also have been deployed at this point. This automates the process of funding the faucet in non-dry run test networks and isolated testing environments like devenv